lxd/storage/drivers/utils_cgo.go

```diff
@@ -265,6 +265,7 @@ import "C"
 // LoFlagsAutoclear determines whether the loop device will autodestruct on last
 // close.
 const LoFlagsAutoclear int = C.LO_FLAGS_AUTOCLEAR
+const LoFlagsDirectIO int = C.LO_FLAGS_DIRECT_IO
```

Comment:
As this is an exported const, it needs to have a comment similar to the `LoFlagsAutoclear` above it, or it will fail `golint`.